src/agent/structures/structures.ts

```diff
@@ -5,39 +5,45 @@ import { Pushes } from "./pushes";
 import { InitialUser, User } from "./users";
 
 export interface Geolocation {
-  country: string;
-  country_code: string;
-  region: string;
-  city: string;
-  timezone: string;
+  country?: string;
+  country_code?: string;
+  region?: string;
+  city?: string;
+  timezone?: string;
+  latitude?: string;
+  longitude?: string;
 }
 
 export interface LastPage {
   opened_at: string;
   url: string;
-  title: string;
+  title?: string;
 }
 
 export interface Visit {
-  started_at: string;
-  ended_at: string;
-  referrer: string;
-  ip: string;
-  user_agent: string;
-  geolocation: Geolocation;
-  last_pages: LastPage[];
+  id: number;
+  started_at?: string;
+  ended_at?: string;
+  referrer?: string;
+  ip?: string;
+  user_agent?: string;
+  geolocation?: Geolocation;
+  last_pages?: LastPage[];
 }
 
 export interface Statistics {
   chats_count: number;
   threads_count: number;
   visits_count: number;
+  page_views_count: number;
+  greetings_shown_count: number;
+  greetings_Accepted_count: number
 }
 
 export interface Chat {
   id: string;
   users: User[];
-  threads: Thread[];
+  thread?: Thread;
```

Comment:
👍 Good point, please change this threads list to single optional thread in all non-deprecated versions (v3.3, v3.4, v3.5, v3.6)